administrator/language/en-GB/com_users.ini

```
@@ -127,9 +127,12 @@ COM_USERS_FIELD_LOGOUT_REDIRECTMENU_LABEL="Menu Item Logout Redirect"
 COM_USERS_FIELD_LOGOUT_REDIRECT_CHOICE_DESC="'Internal URL' lets you manually enter any internal URL in the Redirect field. 'Menu Item' lets you directly select an existing menu item.<br>For a multilingual site, it is recommended to use 'Menu Item'."
 COM_USERS_FIELD_LOGOUT_REDIRECT_CHOICE_LABEL="Choose Logout Redirect Type"
 COM_USERS_FIELD_LOGOUT_REDIRECT_ERROR="Only one of the logout redirect fields should have a value."
+COM_USERS_FIELD_REGISTRATION_LABEL="Registration Page"
+COM_USERS_FIELD_REGISTRATION_DESC="Set a Menu Item if you wish to utilize a custom registration page."
```

```suggestion
COM_USERS_FIELD_REGISTRATION_DESC="Set a Menu Item to utilize a custom registration page."
```
Also, please sort keys in alpha order.